docs/sql-ref-syntax-qry-window.md

```diff
@@ -19,4 +19,208 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+Window functions operate on a group of rows, referred to as a window, and calculate an aggregated value for each row based on the specified window. Window functions are useful for processing tasks such as calculating a moving average, computing a cumulative statistic, or accessing the value of rows given the relative position of the current row. Spark SQL supports three types of window functions:
+
+  * Ranking Functions
+  * Analytic Functions
+  * Aggregate Functions
+
+### Syntax
+
+{% highlight sql %}
+window_function OVER ( [ partition_spec ] order_spec [ window_frame ] )
+{% endhighlight %}
+
+### Parameters
+
+<dl>
+  <dt><code><em>window_function</em></code></dt>
+  <dd>
+    <ul>
+      <li> Ranking Functions </li>
+      <br>
+      <b>Syntax:</b>
+        <code>
+          RANK | DENSE_RANK | PERCENT_RANK | NTILE | ROW_NUMBER
+        </code>
+    </ul>
+    <ul>
+      <li> Analytic Functions </li>
+      <br>
+      <b>Syntax:</b>
+        <code>
+          CUME_DIST | LAG | LEAD
+        </code>
+    </ul>
+    <ul>
+      <li> Aggregate Functions </li>
+      <br>
+      <b>Syntax:</b>
+        <code>
+          MAX | MIN | COUNT | SUM | AVG | ...
+        </code>
+        <br>
+        Please refer <a href="api/sql/">here</a> for a complete list of Spark Aggregate Functions.
```

Comment:
nit: `here` -> `the Built-in Function document`?

Reply:
nit: `Spark Aggregate Functions.` -> `Spark aggregate functions.`?

Reply:
I will put ```sql-ref-functions-builtin.html``` as the link for Built-in Function document. It's broken now but will work after your PR is in.

Reply:
Ur, could you revert the link back? I'm currently not sure that my PR is target at 3.0.